docs/docsite/rst/roadmap/COLLECTIONS_12.rst

```diff
@@ -20,24 +20,30 @@ This release schedule includes dates for the `ansible <https://pypi.org/project/
-:2025-06-03: Ansible-12.0.0rc2 when necessary, otherwise Ansible-12.0.0 release.
-:2025-06-10: Ansible-12.0.0 release when Ansible-12.0.0rc2 was necessary.
-:2025-06-03 or 2023-06-10: Create the ansible-build-data directory and files for Ansible-13.
-:2025-06-16: Release of ansible-core 2.19.1.
-:2025-06-17: Release of Ansible-12.1.0 (bugfix + compatible features: every four weeks.)
-
-.. [1] In case there are any additional ansible-core beta releases or release candidates, we will try to do another Ansible-12.0.0 alpha release. This might mean that we will release Ansible-12.0.0 alpha2 earlier (and release Ansible-12.0.0 alpha3 or later on 2025-04-29) and / or release one or more additional alpha after 2024-04-29.
+    Since `ansible-core 2.19.0's schedule has been delayed <https://forum.ansible.com/t/41504>`__, Ansible 12.0.0's schedule will be adjusted accordingly.
+    The below dates will be updated once we know more about the ansible-core 2.19.0 release schedule.
+
+    The release schedule of Ansible 12.0.0 will follow the :ref:`general Ansible release schedule <ansible_general_major_release_schedule>`, which depends on ansible-core 2.19.0's release schedule as described in that link.
+
+:~~2025-03-31~~: ansible-core feature freeze, stable-2.19 branch created.
+:~~2025-04-07~~: Start of ansible-core 2.19 betas
+:~~2025-04-08~~: Ansible-12.0.0 alpha1 [1]_
+:~~2025-04-28~~: First ansible-core 2.19 release candidate.
+:~~2025-04-29~~: Ansible-12.0.0 alpha2 [1]_
+:~~2025-05-19~~: Ansible-core-2.19.0 released.
+:~~2025-05-19~~: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-12. This includes adding new collections to Ansible 12.0.0; from now on new collections have to wait for 12.1.0 or later.
+:~~2025-05-20~~: Ansible-12.0.0 beta1 -- feature freeze [2]_ (collection owners and interested users should test for bugs).
+:~~2025-05-27~~: Ansible-12.0.0 rc1 [3]_ [4]_ (weekly release candidates as needed; test and alert us to any blocker bugs). Blocker bugs will slip release.
+:~~2025-05-30~~: Last day to trigger an Ansible-12.0.0rc2 release because of major defects in Ansible-12.0.0rc1.
+:~~2025-06-03~~: Ansible-12.0.0rc2 when necessary, otherwise Ansible-12.0.0 release.
+:~~2025-06-10~~: Ansible-12.0.0 release when Ansible-12.0.0rc2 was necessary.
+:~~2025-06-03 or 2023-06-10~~: Create the ansible-build-data directory and files for Ansible-13.
+:~~2025-06-16~~: Release of ansible-core 2.19.1.
+:~~2025-06-17~~: Release of Ansible-12.1.0 (bugfix + compatible features: every four weeks.)
+
+.. [1] In case there are any additional ansible-core beta releases or release candidates, we will try to do another Ansible-12.0.0 alpha release. This might mean that we will release Ansible-12.0.0 alpha2 earlier (and release Ansible-12.0.0 alpha3 or later on ~~2025-04-29~~) and / or release one or more additional alpha after ~~2025-04-29~~.
```

Comment:
Now that #2497 isn't a draft anymore, maybe we should wait a bit until it's merged and give some definite dates here.

Reply:
I don't think we should leave the wrong dates there much longer.

Reply:
#2497 has been merged, so we can update the dates accordingly :-)